package.json

```diff
@@ -51,22 +51,21 @@
     "./esm/priorities": "./lib/constants/priorities.mjs",
     "./esm/support": "./lib/support/index.cjs"
   },
-  "main": "lib/index.js",
+  "main": "dist/lib/index.js",
   "bin": {
-    "jhipster": "cli/jhipster.js"
+    "jhipster": "dist/cli/jhipster.js"
   },
   "files": [
-    "cli",
-    "generators",
-    "jdl",
-    "lib",
-    "utils",
+    "dist",
     "!**/__*",
     "!**/*.snap",
     "!**/*.spec.?(c|m)js"
   ],
   "scripts": {
+    "build": "npm run compile && npm run copy-files",
+    "compile": "tsc",
     "completion": "tabtab install --name jhipster --auto",
+    "copy-files": "cpy 'generators/**' '!**/*.(js|ts|cjs|mjs)' dist/generators && cpy 'package.json' dist",
```

Comment:
Some modules include the package.json and I think it's easier/cleaner to also copy it to `dist` so that every file needed to run the program is in `dist` without making an exception for the package.json.

Reply:
I am not sure I understand. These files include the package.json: 
![image](https://user-images.githubusercontent.com/39118431/190001118-dd8fb95d-5bb8-4a27-900f-6fb0e446070c.png)
(I blacked out files that include another package.json, not the one we are talking about)

Do you propose to not copy the `package.json`?